app/src/main/java/ai/elimu/launcher/MainActivity.java

```diff
@@ -21,18 +24,25 @@ protected void onCreate(Bundle savedInstanceState) {
         try {
             PackageInfo packageInfoAppstore = getPackageManager().getPackageInfo(BuildConfig.APPSTORE_APPLICATION_ID, 0);
             Timber.i( "packageInfoAppstore.versionCode: " + packageInfoAppstore.versionCode);
+
+            Intent intent = new Intent(getApplicationContext(), HomeScreensActivity.class);
+            startActivity(intent);
+
         } catch (PackageManager.NameNotFoundException e) {
             Timber.w( null, e);
             Toast.makeText(getApplicationContext(), "The elimu.ai Launcher will not work without the elimu.ai Appstore: " + BuildConfig.APPSTORE_APPLICATION_ID, Toast.LENGTH_LONG).show();
-            // TODO: Add link to GitHub for downloading APK
-        }
 
-        Intent intent = new Intent(getApplicationContext(), HomeScreensActivity.class);
-        startActivity(intent);
+            goToAppstoreDownload();
```

Comment:
@gscdev It's a bit confusing to be redirected to a URL without any form of warning about what's happening. Maybe we could use a [snackbar](https://material.io/develop/android/components/snackbars) (or dialog) with an action button for this?

Reply:
Something like this?

![Screenshot from 2020-09-22 01-13-12](https://user-images.githubusercontent.com/8291318/93830746-5cffe680-fc71-11ea-8995-5197699a1e44.png)

Reply:
@gscdev Yes, perfect :-)